lib/cfa/grub2/default.rb

```diff
@@ -140,7 +140,7 @@ def terminal=(values)
       end
 
       def serial_console=(value)
-        self.terminal = :serial
+        self.terminal = [:serial]
```

Comment:
with respect to new API I am not sure if it is right behavior, maybe it should just add `:serial` if it is not already. What do you think?